### PR TITLE
Add 1.30&1.31 to kind e2e presubmits and remove 1.26&1.27

### PR DESCRIPTION
**Pull request by liangyuanpeng**
~~for easy to review,i have not resort the jobs, so it is:~~

- 1.29
- 1.28
- 1.31
- 1.30

and please let me know if we need to sort it.

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test

you can add a new commit now just sorting them, 

Thank you very much

File: config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml [hunk 1]
```diff
@@ -305,53 +305,50 @@ presubmits:
...
     - org: kubernetes
       repo: kubernetes
-      base_ref: release-1.28
+      base_ref: release-1.30
       path_alias: k8s.io/kubernetes
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    name: pull-kind-e2e-kubernetes-1-30
+    cluster: k8s-infra-prow-build
```
[the remark this answers is not in the document]
Comment (liangyuanpeng) [reply config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml]: I just need to change the version number without moving the block, now it should be easy enough to review!

[the remark this answers is not in the document]
Comment (liangyuanpeng) [reply config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml]: > if we can move this to the beginning it will make easier to read

also update for it

**Comment by liangyuanpeng** [comment]
```shell
...
config/prow/cluster/build/build_kubernetes-external-secrets_deployment.yaml
  39:11     warning  comment not indented like content  (comments-indentation)
ERROR: yamllint failed
verifying rollup typescripts
Rollup gopherage/cmd/html/static/browser.ts
failed to create storage client: dialing: google: could not find default credentials. See https://cloud.google.com/docs/authentication/external/set-up-adc for more information
WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
...
```


verify-lint is working on my local, let met try to retest it.

```shell
$ hack/make-rules/verify/all.sh
...
go: downloading github.com/jmhodges/clock v1.2.0
go: downloading github.com/go-openapi/analysis v0.21.4
go: downloading github.com/go-openapi/jsonpointer v0.20.0
go: downloading github.com/go-openapi/loads v0.21.2
SUCCESS: ran go mod tidy in root and hack/tools folders

All verify checks passed, congrats!

```

**Comment by aojea** [comment]
/lgtm
/approve

Thank you very much

**Pull request merged**